scripts/aggregate-themes/README.md

```diff
@@ -18,22 +17,27 @@ project
 
 Within this file declare and export an object containing your theme configuration.
 
-This configuration will be used to aggregate nested dependency themes and output a single `aggregated-themes.js` file. This file will be automatically included as an entry point within your application if you are using the webpack configuration provided by terra-toolkit.
+Theme aggregation uses this configuration to search for theme files within the following directory format: `theme->themeName->themeName.scss`. Then, for each theme, a root and/or scope themes will be generated.
```

Comment:
nits: Themes are normally hyphenated (for themeName). Is the starting `theme-` a typo?

 `project/themes/terra-example-theme/terra-example-theme.scss`

Reply:
typo :( https://github.com/cerner/terra-toolkit/pull/324/commits/0a976941afbe295532f1f4b742f97462fdcfe2a6